x/nft/simulation/operations.go

```diff
@@ -0,0 +1,186 @@
+package simulation
+
+import (
+	"errors"
+	"fmt"
+	"math/rand"
+
+	"github.com/tendermint/tendermint/crypto"
+
+	"github.com/cosmos/cosmos-sdk/baseapp"
+	"github.com/cosmos/cosmos-sdk/simapp/helpers"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/nft/internal/keeper"
+	"github.com/cosmos/cosmos-sdk/x/nft/internal/types"
+	"github.com/cosmos/cosmos-sdk/x/simulation"
+)
+
+// DONTCOVER
+
+// SimulateMsgTransferNFT simulates the transfer of an NFT
+func SimulateMsgTransferNFT(ak types.AccountKeeper, k keeper.Keeper) simulation.Operation {
+	return func(r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
+		accs []simulation.Account) (opMsg simulation.OperationMsg, fOps []simulation.FutureOperation, err error) {
+
+		ownerAddr, denom, nftID := getRandomNFTFromOwner(ctx, k, r)
+		if ownerAddr.Empty() {
+			return simulation.NoOpMsg(types.ModuleName), nil, nil
+		}
+
+		msg := types.NewMsgTransferNFT(
+			ownerAddr,                             // sender
+			simulation.RandomAcc(r, accs).Address, // recipient
+			denom,
+			nftID,
+		)
+
+		acc, found := simulation.FindAccount(accs, ownerAddr)
```

Comment:
this seems redundant, isn't `acc == fromAcc`? or is this just a sanity check that the owner acc is a sim account?

Reply:
`fromAcc` is `exported.Account`, whereas `acc` is a `simulation.Account`